src/dynamic_boundary_conditions/tide/sea_level_rise_data.py

```diff
@@ -3,132 +3,27 @@
-        driver_location = subprocess.check_output("which geckodriver", shell=True,
-                                                  stderr=subprocess.STDOUT).decode().strip()
-        # Create firefox with explicit driver_location since selenium does not always find the correct driver binary
-        firefox_service = webdriver.FirefoxService(executable_path=driver_location)
-        # Create firefox webdriver
-        driver = webdriver.Firefox(options=firefox_options, service=firefox_service)
-
-    # Open the specified website in the browser
-    driver.get("https://searise.takiwa.co/map/6245144372b819001837b900")
-    # Add a 1-second delay to ensure that the website is open before downloading
-    time.sleep(1)
-    # Find and click the "Accept" button on the web page
-    driver.find_element(By.CSS_SELECTOR, "button.ui.green.basic.button").click()
-    # Find and click "Download"
-    driver.find_element(By.ID, "container-control-text-6268d9223c91dd00278d5ecf").click()
-    # Find and click "Download Regional Data"
-    [element.click() for element in driver.find_elements(By.TAG_NAME, "h5") if element.text == "Download Regional Data"]
-    # Identify links to all the regional data files on the webpage
-    elements = driver.find_elements(By.CSS_SELECTOR, "div.content.active a")
-    # Iterate through the identified links and simulate a click action to trigger the download
-    for element in elements:
-        # Scroll down the div to the link. Required for firefox browser
-        driver.execute_script("arguments[0].scrollIntoView(true);", element)
-        # Click the download link
-        ActionChains(driver).move_to_element(element).click().perform()
-        # Wait a short delay before clicking again so that the download starts.
-        time.sleep(0.5)
-    # Add a 5-second delay to ensure all downloads are complete before quitting the browser
-    time.sleep(5)
-    # Quit the WebDriver, closing the browser
-    driver.quit()
-    # If running this from windows within a WSL directory, Zone.Identifier files are created and must be removed.
-    for zone_identifier_file in slr_data_dir.glob("*Zone.identifier"):
-        os.remove(zone_identifier_file)
-    # Check that the number of downloaded files matches the number of links on the webpage
-    slr_dir_files = list(slr_data_dir.glob("*"))
-    if len(slr_dir_files) != len(elements):
-        logging.debug(f"slr_dir_files = {slr_dir_files}")
-        logging.debug(f"elements = {elements}")
-        raise ValueError(f"The number of files in slr_data_dir ({len(slr_dir_files)})"
-                         f" does not match the number of datasets found on the web page ({len(elements)})")
-    # Log that the files have been successfully downloaded
-    log.info("Successfully downloaded regional 'sea_level_rise' data files from NZ SeaRise Takiwa.")
-
-
-def read_slr_data_from_files(slr_data_dir: pathlib.Path) -> gpd.GeoDataFrame:
+def read_slr_data_from_files(
+) -> gpd.GeoDataFrame:
```

Comment:
```suggestion
def read_slr_data_from_takiwa() -> gpd.GeoDataFrame:
```
Its appropriate to mention the original source of the data. And we don't read it from files so that isn't quite right. This will need some changes in the uses of this function.